posixlib/src/main/scala/scala/scalanative/posix/unistd.scala

```diff
@@ -17,6 +17,7 @@ object unistd {
   def dup2(fildes: CInt, fildesnew: CInt): CInt   = extern
   def execve(path: CString, argv: Ptr[CString], envp: Ptr[CString]): CInt =
     extern
+  def execvp(path: CString, args: Ptr[CString]): CInt             = extern
```

Comment:
I have an outstanding PR #1622 "WIP Fix #1621: supply complete POSIX unistd definitions" which
addresses some of these issues. We may need to manage merge conflicts.  No problem. 

I noticed that execv() seems to be missing in this edit.  The GNU execvpe() is an open question. Do we
be useful & non-Posix or strictly Posix?  

In checking the Open Group 2018 specification for this comment, I discovered that unistd.h should
declare `fexecve(int, char *const [], char *const []);`.

Reply:
@LeeTibbert the feedback I got from Denys is that only standardized POSIX functions belong in posixlib.  

I'll add `execv()` in this PR - less eager to implement `fexecve` since I've never used it, and wouldn't really know how to test it.

Reply:
re: execvpe Understood and agreed.   A GNUism.

re: fexecve.  I believe that unit tests for straight "passthre" methods are encouraged but
not strictly required.  I think the bar is a bit higher for those where the scala "glue" layer
does something, like handling impedance between SN and C structures.

I could not find an existing UnistdSuite.scala, so I think the lack of a fexecve() (or others
from this PR) is fine.  One voice, others may differ.

Reply:
OK by me if fexecve() is delayed until my PR.  May motivate me.
I was just wearing my PR review hat and pointing out inconsistencies.
Can't fix the whole world at once.